_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: OTel metrics raw
```

Comment:
@carolxob, Raj indicated that we're renaming this processor to Otel metrics (removing raw). I told him to comment but looks like he hasn't.

Reply:
@lizsnyder Thanks for letting me know! I'll make sure this change is incorporated.